curriculum/challenges/english/25-front-end-development/lab-nth-fibonacci-number-generator/66d9af3897e7d75a895b72c2.md

```diff
@@ -0,0 +1,116 @@
+---
+id: 66d9af3897e7d75a895b72c2
+title: Build the nth Fibonacci number generator
+challengeType: 14
+dashedName: lab-nth-fibonacci-number-generator
+---
+
+# --description--
+
+In this lab you will use a dynamic programming approach to compute and return the `n`-th number from the Fibonacci sequence, in which each number is the sum of the two preceding numbers.
+
+**Objective:** Fulfill the user stories below and get all the tests to pass to complete the lab. 
+
+**User Stories:**
+
+1. You should define an array named `sequence` in the global scope. It should contain the number `0` and `1`.
+1. You should create a function named `fibonacci`.
+1. The `fibonacci` function should take a single parameter, which is a positive integer.
+1. If the input is not a positive integer, the function should return `"Invalid input"`.
```

Comment:
The code also checks if the type is not of type number, this should be described in the user story.